src/lib/build-flags.ts

```diff
@@ -1,2 +1,4 @@
 export const LOGIN_INCLUDE_DEV_SERVERS = true
 export const PWI_ENABLED = true
+// TODO
+export const NEW_ONBOARDING_ENABLED = true
```

Comment:
TODO decide how we want to ship this

Reply:
_probably_ won't need this but let's make sure our server routes get shipped before we remove it